src/etc/cargo.1

```diff
@@ -1,4 +1,4 @@
-.TH CARGO "1" "September 2014" "cargo 0.0.1-pre" "User Commands"
+.TH CARGO "1" "November 2015" "cargo 0.7.0" "User Commands"
```

Comment:
What should be the correct values here since the changes will only appear in a future version?! Should I leave it as before?